messages/bg.json

```diff
@@ -5,6 +5,14 @@
   "admin/b2b-quotes.settings.saveSettings.failure": "Неуспешно запазване на настройките",
   "admin/b2b-quotes.settings.saveSettings.success": "Настройките са запазени успешно",
   "admin/b2b-quotes.settings.cartLifeSpan.label": "Период на изтичане по подразбиране (в дни) за оферти и запазени колички",
+  "admin/b2b-quotes.settings.saveSettings.management.quotes.experiration.data": "Expiration Period for Quotes and Saved Carts",
```

Comment:
There's a typo in the property name. It is written `experiration` and should be `expiration`. The same issue is present in the messages in other languages. Also, this property text is in English in all languages and not translated.